config/projects/datadog-agent.rb

```diff
@@ -193,7 +193,7 @@
 dependency 'psutil'
 dependency 'requests'
 
-if !ENV["USE_INTEGRATION_SDK"]
+unless ENV["USE_INTEGRATION_SDK"] == 'true'
```

Comment:
yea checking for nil was too buggy, this was much better, I'll update the description and then merge it